bin/mastarm-deploy

```diff
@@ -108,18 +116,40 @@ async function deploy () {
     :hash: *commit:* ${commit}
     :seedling: *env:* ${env}
     :compression: *minify:* ${minify}
-    :package: *s3bucket:* ${s3bucket}
-    :hammer_and_wrench: *building:* ${sourceFiles.join(', ')}`
+    :package: *s3bucket:* ${s3bucket}`
```

Comment:
I think some of the above could be refactored a bit to be a little more readable. For example, `buildOpts` is only used when a `staticFileDirectory` is missing, so that could be moved into the `else` block below. This is a bit hard to describe with words, so I have a PR incoming to suggest changes.